src/fps/example/task-00/opmainnetConfig.toml

```diff
@@ -0,0 +1,6 @@
+# this is the file used to determine the network configuration
+
+l2chains = [{name = "Optimism Mainnet", chainId = 10}]
```

Comment:
I don't think I follow why we have this called `opmainnetConfig.toml`. Based on the conversation we had around the fact these config files will live in directories based on the network e.g. `tasks/eth` isn't this redundant?